### PR TITLE
Added Test for updating a field and adding a new field

### PR DESCRIPTION
**Pull request by Abyhsen**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -42,8 +42,8 @@
 	},
 	"minimum-stability": "dev",
 	"scripts": {
-        "post-install-cmd": "./vendor/bin/cghooks add --ignore-lock",
-        "post-update-cmd": "./vendor/bin/cghooks update",
+        "post-install-cmd": "sh ./vendor/bin/cghooks add --ignore-lock",
+        "post-update-cmd": "sh ./vendor/bin/cghooks update",
```
Comment (berni2288) [inline composer.json]: Windows does not have "sh", why is this necessary?

Comment (Abyhsen) [reply composer.json]: If you use git bash there's a sh command and the hooks are added/updated after composer install/update.
Sadly ./ alone doesn't suffice.
